<launch>
    <group ns="Drive">
        <node name="Drivetrain_Control" pkg="mavric" type="SparkCAN_Drive_Train.py" >
            <remap from="Drive_Train" to="Drive_Command" />
            <remap from="Steer_Train" to="Steer_Command" />
        </node>
        <node name="Drivetrain_Interface" pkg="mavric" type="Drive_Train_P.py" respawn="true" >
            <remap from="Drive_Train" to="Drive_Command" />
            <remap from="Steer_Train" to="Steer_Command" />
            <remap from="Steer_Cal" to="Cal_Command" />
            <param name="port" value="9002" />
        </node>
        <node name="Drivetrain_Heartbeat" pkg="mavric" type="DrivetrainHeartbeat.py" respawn="true" >
            <param name="timeout" value="1" />
            <remap from="Drive_Train" to="Drive_Command" />
        </node>
    </group>
    <group ns="Arm">
        <node name="Armtrain" pkg="mavric" type="mavric_CAN_Arm_Train" >
          <param name="sr_ramp_rate_up" value="300"/>
          <param name="sr_ramp_rate_down" value="400"/>
          <param name="ep_ramp_rate_up" value="300"/>
          <param name="ep_ramp_rate_down" value="400"/>
          <param name="wr_ramp_rate_up" value="300"/>
          <param name="wr_ramp_rate_down" value="400"/>
          <param name="wp_ramp_rate_up" value="300"/>
          <param name="wp_ramp_rate_down" value="400"/>
          <param name="sr/Scale" value=".1"/>
          <param name="ep/Scale" value=".1"/>
          <param name="wr/Scale" value=".1"/>
          <param name="wp/Scale" value=".1"/>
        </node>
        <node name="Arm_Interface" pkg="mavric" type="Arm_P.py" respawn="true">
            <param name="port" value="10001" />
        </node>
    </group>
</launch>